### PR TITLE
[WIP][ENH] mne_setup_forward_model command

### PR DESCRIPTION
**Pull request by vferat**
## Reference issue

Part of discussion in #6707
## What does this implement/fix?

This PR adds a command line function **mne_setup_forward_model** that wraps **mne.make_bem_model** and save its ouput using **mne.write_bem_surfaces**.

It tries to replicate the [MNE-C API](https://mne.tools/0.17/manual/c_reference.html#mne-setup-forward-model).

### CODE REVIEW

**Comment by agramfort** [comment]
don't forget to update what's new page. Travis complains

**Comment by vferat** [comment]
Spotted differences between the [MNE-C API](https://mne.tools/0.17/manual/c_reference.html#mne-setup-forward-model) and [MNE-Python API](https://mne.tools/stable/generated/mne.make_bem_model.html)

MNE-Python:

- no **--noswap** option

- no **--innershift** or **--innershift** options

- no **--innershift**, **--outershift**, **--scalpshift** options

Is there a need/way to implement it in python ?

In MNE-C the **--homog** flag does not take any conductivity, in MNE-Python a conductivity is still needed for a single layer model. Should we accept other values than 0.3 (default to MNE-C)?

File: mne/commands/mne_setup_forward_model.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    brainc = options.brainc
+    skullc = options.skullc
+    scalpc = options.scalpc
+    homog = True if options.homog is not None else False
+    verbose = True if options.verbose is not None else False
+
+    # Parse conductivity option
+    if homog is True:
+        # Single layer
+        conductivity = [0.3]
```
Comment (agramfort) [inline mne/commands/mne_setup_forward_model.py]: use brainc not hard code 0.3

Comment (vferat) [reply mne/commands/mne_setup_forward_model.py]: Can I use something like --homog [number] instead ? I think that it's less misleading than using both --brainc and --homog.

**Comment by agramfort** [comment]
no as brainc is what 0.3 corresponds to. it's the conductivity of the brain

>

File: mne/commands/mne_setup_forward_model.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                      help='Defines the scalp compartment conductivity. '
+                           'The default value is 0.3 S/m.',
+                      default=None,
+                      type='float')
+    parser.add_option('--homog',
+                      dest='homog',
+                      help='Use a single compartment model (brain only) '
+                           'instead a three layer one (scalp, skull, and '
+                           ' brain). If this flag is specified, the options '
+                           '--brainc, --skullc, and --scalpc are irrelevant.',
```
Comment (agramfort) [inline mne/commands/mne_setup_forward_model.py]: to fix

**Pull request merged**